magstock/templates/static_views/legal_name.html

```diff
@@ -0,0 +1,20 @@
+<html>
```

Comment:
In general we need a much better way to configure text inside templates like this. Not every text change can be solved by making the original more generic. If/when we can convert this to a django app, we'll have some options open up.

But for now, I'm just going to fix this by changing the original template.